podcasts.xcodeproj/project.pbxproj

```diff
@@ -8550,9 +8550,9 @@
 				BD5231EA1E3EDA6A000F0D38 /* Embed App Extensions */,
 				BDD3018A1EFB9356004A9972 /* Embed Watch Content */,
 				40BF0B3F241F55FE003E3ABD /* Embed Frameworks */,
-				C716FF45296F2BE3006B787D /* Modify Plist */,
-				CD7B4456FFD51509B59727FB /* [CP] Embed Pods Frameworks */,
 				9AC09DDA2D2D4103003EBA96 /* Embed App Clips */,
+				CD7B4456FFD51509B59727FB /* [CP] Embed Pods Frameworks */,
+				C716FF45296F2BE3006B787D /* Modify Plist */,
```

Comment:
This is what addressed the cycle that I reported in https://github.com/Automattic/pocket-casts-ios/pull/2598

Reply:
Nice find!